Model/Maker/AbstractMaker.php

```diff
@@ -382,6 +397,21 @@ protected function makeJsMixinFilenameQuestion(string $webArea): Question
         return $question;
     }
 
+    protected function makeGridNamespaceQuestion(
+        \Ctasca\MageBundle\Model\Template\Locator $templateLocator,
+        string $string
+    )
+    {
+        $question = $this->questionFactory->create(
+            'Enter the namespace for the ui component' .
+            "\n<comment>The name of xml will be {{namespace}}_grid \n" .
+            "File will be created in the Company/Module/view/adminhtml/ui-component directory" .
```

Comment:
File will be created in the Company/Module/view/adminhtml/ui-component directory. Dynamically set path depending on area choice question